docs/code-examples/annotation_context_connections.py

```diff
@@ -1,28 +1,37 @@
 import rerun as rr
+import rerun.experimental as rr2
+from rerun.experimental import dt as rrd
```

Comment:
```suggestion
from rerun.experimental import dt as rrd
```

I think we should avoid `rrd` as the convention here due to name collision with `rrd` the file extension.

Reply:
https://github.com/rerun-io/rerun/issues/3236